core/db/migrate/20130807024301_upgrade_adjustments.rb

```diff
@@ -14,7 +14,8 @@ def up
 
     # Tax adjustments have their sources altered
     Spree::Adjustment.where(:originator_type => "Spree::TaxRate").find_each do |adjustment|
-      adjustment.source = adjustment.originator
+      adjustment.source.id = adjustment.originator_id
+      adjustment.source.type = 'Spree::TaxRate'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.